### PR TITLE
add --check to the output of run.sh --help.

### PR DESCRIPTION
**Pull request by TingluoHuang**

### CODE REVIEW

**Review by lokesh755 — APPROVED** [review]

File: src/Runner.Listener/Runner.cs [hunk 1]
```diff
@@ -510,14 +511,17 @@ private void PrintUsage(CommandSettings command)
  --runnergroup string   Name of the runner group to add this runner to (defaults to the default runner group)
  --labels string        Extra labels in addition to the default: 'self-hosted,{Constants.Runner.Platform},{Constants.Runner.PlatformArchitecture}'
  --work string          Relative runner work directory (default {Constants.Path.WorkDirectory})
- --replace              Replace any existing runner with the same name (default false)");
+ --replace              Replace any existing runner with the same name (default false)
+ --pat                  GitHub personal access token used for checking network connectivity when execute `.{separator}run.{ext} --check`");
```
Comment (juliobbv) [inline src/Runner.Listener/Runner.cs:515]: Minor: `when executing`

Comment (TingluoHuang) [reply src/Runner.Listener/Runner.cs]: fixed.

**Review by ericsciple — APPROVED** [review]

File: src/Runner.Listener/Runner.cs [hunk 2]
```diff
@@ -510,14 +511,17 @@ private void PrintUsage(CommandSettings command)
  --runnergroup string   Name of the runner group to add this runner to (defaults to the default runner group)
  --labels string        Extra labels in addition to the default: 'self-hosted,{Constants.Runner.Platform},{Constants.Runner.PlatformArchitecture}'
  --work string          Relative runner work directory (default {Constants.Path.WorkDirectory})
- --replace              Replace any existing runner with the same name (default false)");
+ --replace              Replace any existing runner with the same name (default false)
+ --pat                  GitHub personal access token used for checking network connectivity when executing `.{separator}run.{ext} --check`");
```
Comment (ericsciple) [inline src/Runner.Listener/Runner.cs:515]: Should this should the `{separator}run{ext}` - i.e. remove the `.`?

Comment (ericsciple) [reply src/Runner.Listener/Runner.cs:515]: same question further below on line 523

Comment (TingluoHuang) [reply src/Runner.Listener/Runner.cs:515]: we have the `.` in everywhere else. 😄

**Pull request merged**